[2023-11-23T20:42:43.297+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: COVID-19.Data_transform_and_load manual__2023-11-22T22:30:11.481856+00:00 [queued]>
[2023-11-23T20:42:43.304+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: COVID-19.Data_transform_and_load manual__2023-11-22T22:30:11.481856+00:00 [queued]>
[2023-11-23T20:42:43.305+0000] {taskinstance.py:1361} INFO - Starting attempt 4 of 4
[2023-11-23T20:42:43.315+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): Data_transform_and_load> on 2023-11-22 22:30:11.481856+00:00
[2023-11-23T20:42:43.320+0000] {standard_task_runner.py:57} INFO - Started process 788 to run task
[2023-11-23T20:42:43.321+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'COVID-19', 'Data_transform_and_load', 'manual__2023-11-22T22:30:11.481856+00:00', '--job-id', '82', '--raw', '--subdir', 'DAGS_FOLDER/dag_etl.py', '--cfg-path', '/tmp/tmpu18np0ub']
[2023-11-23T20:42:43.323+0000] {standard_task_runner.py:85} INFO - Job 82: Subtask Data_transform_and_load
[2023-11-23T20:42:43.354+0000] {task_command.py:416} INFO - Running <TaskInstance: COVID-19.Data_transform_and_load manual__2023-11-22T22:30:11.481856+00:00 [running]> on host 89d301135b3a
[2023-11-23T20:42:43.549+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='COVID-19' AIRFLOW_CTX_TASK_ID='Data_transform_and_load' AIRFLOW_CTX_EXECUTION_DATE='2023-11-22T22:30:11.481856+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-11-22T22:30:11.481856+00:00'
[2023-11-23T20:42:43.757+0000] {warnings.py:109} WARNING - /opt/***/dags/dag_etl.py:55: FutureWarning: section/key [core/sql_alchemy_conn] has been deprecated, you should use[database/sql_alchemy_conn] instead. Please update your `conf.get*` call to use the new name
  conn_uri = conf.get('core', 'sql_alchemy_conn')

[2023-11-23T20:42:43.788+0000] {python.py:194} INFO - Done. Returned value was:     num_departement          nom_departement                  nom_region
0                01                      Ain        Auvergne-Rhône-Alpes
1                02                    Aisne             Hauts-de-France
2                03                   Allier        Auvergne-Rhône-Alpes
3                04  Alpes-de-Haute-Provence  Provence-Alpes-Côte d'Azur
4                05             Hautes-Alpes  Provence-Alpes-Côte d'Azur
..              ...                      ...                         ...
96              971               Guadeloupe                  Guadeloupe
97              972               Martinique                  Martinique
98              973                   Guyane                      Guyane
99              974               La Réunion                  La Réunion
100             976                  Mayotte                     Mayotte

[101 rows x 3 columns]
[2023-11-23T20:42:43.819+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=COVID-19, task_id=Data_transform_and_load, execution_date=20231122T223011, start_date=20231123T204243, end_date=20231123T204243
[2023-11-23T20:42:43.855+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-23T20:42:43.869+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
